code/API_definitions/CAMARA/number_verification.yaml

```diff
@@ -188,13 +188,14 @@ components:
         - message
       properties:
         status:
-          type: string
-          pattern: '^[1-5][0-9][0-9]$'
+          type: integer
+          minimum: 400
+          maximum: 599
           description: HTTP response status code
         message:
           type: string
           description: A human readable description of what the event represent
-    NotAuthenticatedByIpPermissionDenied:
+    PhoneNumberVerificationPermissionDenied:
```

Comment:
The more generic error descriptions are there, the better, so this change is good.